tests/cypress/e2e/Asset/custom_fields.cy.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+describe("Custom Assets - Custom Fields", () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it('Custom field creation and display', () => {
+        // This can be split into multiple tests when the new API supports custom assets and custom fields
+        cy.visit('/front/asset/assetdefinition.form.php');
+        const asset_name_chars = 'abcdefghijklmnopqrstuvwxyz';
+        const asset_name = 'customasset' + Array.from({ length: 10 }, () => asset_name_chars.charAt(Math.floor(Math.random() * asset_name_chars.length))).join('');
+        cy.findByLabelText(/System name/i).type(asset_name);
+        cy.findByLabelText("Active").select('1', { force: true });
+        cy.findByRole('button', {name: "Add"}).click();
+
+        cy.get('div.toast-container .toast-body a').click();
+        cy.url().should('include', '/front/asset/assetdefinition.form.php?id=');
+
+        cy.findByRole('tab', {name: 'Custom fields'}).click();
+        cy.findByRole('button', {name: 'Add a new field'}).parents('.tab-pane').should('have.class', 'active').first().within(() => {
+            cy.findByRole('button', {name: 'Add a new field'}).click();
+            cy.findByLabelText('Label').type('Test String');
+            cy.findByLabelText('Type').select('String', { force: true });
+            cy.findByRole('button', {name: 'Add'}).click();
+        });
+        cy.findByRole('tab', {name: 'Custom fields'}).click();
+        cy.findByRole('button', {name: 'Add a new field'}).parents('.tab-pane').should('have.class', 'active').first().within(() => {
+            cy.findByRole('button', {name: 'Add a new field'}).click();
+            cy.findByLabelText('Label').type('Test Text');
+            cy.findByLabelText('Type').select('Text', { force: true });
```

Comment:
Can use `getSelectByLabel`+ `setDropdownValue` to avoid using `force: true`  (same for other instances).